jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc

```diff
@@ -0,0 +1,102 @@
+= WFLY-14559 Add six new RESTEasy context parameters
+:author:            Ron Sigal
+:email:             rsigal@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Since the RESTEasy context parameters were first exposed in the wildfly/jaxrs management model
+(https://github.com/wildfly/wildfly-proposals/blob/main/jaxrs/WFLY-12298_Change_RESTEASY_settings.adoc),
+several new parameters have been introduced, of which the following should be exposed to the user:
+
+ * resteasy.match.cache.enabled:                       [RESTEASY-2646]
+ * resteasy.match.cache.size:                          [RESTEASY-2646]
+ * resteasy.original.webapplicationexception.behavior: [RESTEASY-2782]
+ * resteasy.patch.filter.disabled:                     [RESTEASY-2280]
+ * resteasy.patch.filter.legacy:                       [RESTEASY-2849]
+ * resteasy.proxy.implement.all.interfaces:            [RESTEASY-2866]
```

Comment:
Should we drop the `resteasy` prefix from the model attribute names? I know most of them have that prefix now, but it's not strictly required and saves some typing for administators.